### PR TITLE
release(v0.9.0): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v0.9.0 release.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,461 @@
...
+* ECDSA Keys for Kubernetes PKI
+* Disk Encryption
+* Virtual Shared IP for Control Plane Endpoint
+
+More in the [docs](https://www.talos.dev/docs/v0.9/introduction/what-is-new/).
+
+
+### Upgrading
+
+Please read [upgrade notes](https://www.talos.dev/docs/v0.9/guides/upgrading-talos/#upgrading-from-talos-08) before upgrading from Talos 0.8.
```
Comment (andrewrynhard) [inline CHANGELOG.md:22]: Please read the ...

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,461 @@
...
+* Artem Chernyshev
+* Alexey Palazhchenko
+* Andrew Rynhard
+* Spencer Smith
+* Seán C McCord
+* Andrew Rynhard
+* Brandon McNama
+* Guilhem Lettron
+* Willem Monsuwe
+* vlad doster
```
Comment (andrewrynhard) [inline CHANGELOG.md:37]: Sir we capitalize this name?

**Review by andrewrynhard — APPROVED** [review]
🚀

File: CHANGELOG.md (hunk 1 above)
Comment (smira) [reply CHANGELOG.md:22]: :facepalm: I'll get it fixed

File: CHANGELOG.md (hunk 2 above)
Comment (smira) [reply CHANGELOG.md:37]: it's going from `git`, so whatever he thinks his name is, it's automated fully. we could use `.mailmap` to do such aggregations in the future

**Review by andrewrynhard — APPROVED** [review]

**Pull request merged**